api/logs/logSender.go

```diff
@@ -103,7 +105,8 @@ func (ls *logSender) waitForPatternMessage() error {
 func (ls *logSender) monitorConnection() {
 	for {
 		mt, _, err := ls.conn.ReadMessage()
-		if mt == websocket.CloseMessage {
+		ls.log.Info("message type", "value", mt)
```

Comment:
Maybe make this logger message of type Trace

Reply:
Done.